### PR TITLE
Fix broken SPP link on popup

### PR DESCRIPTION
**Pull request by jdtoombs**
Now gets project route and changes link on project number accordingly 
brokenlink

### CODE REVIEW

File: frontend/src/components/maps/ParcelPopupView.tsx [hunk 1]
```diff
@@ -21,6 +24,18 @@ export const ParcelPopupView = (props: IParcelDetailProps | null) => {
   const parcelDetail: IParcel | null | undefined = props?.parcel;
   const keycloak = useKeycloakWrapper();
   const location = useLocation();
+  const dispatch = useDispatch();
+  const [projectRoute, setProjectRoute] = useState('');
+
+  useEffect(() => {
+    if (parcelDetail?.projectNumber) {
+      (dispatch(fetchProject(parcelDetail?.projectNumber as string)) as any).then(
```
Comment (Fosol) [inline frontend/src/components/maps/ParcelPopupView.tsx:32]: When the popup is shown it makes a request to get the appropriate route?

**Review by Fosol — APPROVED** [review]

File: frontend/src/components/maps/ParcelPopupView.tsx (hunk 1 above)
Comment (jdtoombs) [reply frontend/src/components/maps/ParcelPopupView.tsx:32]: Yup, gets the project information (if it project number exists) when the user clicks the marker and sets the SPP-# link to the appropriate route

File: frontend/src/components/maps/ParcelPopupView.test.tsx [hunk 2]
```diff
@@ -20,6 +25,10 @@ jest.mock('@react-keycloak/web');
   },
 });
 
+const store = mockStore({
+  [reducerTypes.ProjectReducers.PROJECT]: { stus: { route: `/projects/erp` } },
```
Comment (agalbrai) [inline frontend/src/components/maps/ParcelPopupView.test.tsx:29]: stus?

**Review by agalbrai — APPROVED** [review]

File: frontend/src/components/maps/ParcelPopupView.test.tsx (hunk 2 above)
Comment (jdtoombs) [reply frontend/src/components/maps/ParcelPopupView.test.tsx:29]: good eye

Comment (agalbrai) [reply frontend/src/components/maps/ParcelPopupView.test.tsx:29]: I just didn't know what it was lol

**Pull request merged**